src/Core/Defines.h

```diff
@@ -63,6 +63,7 @@
 
 #define DBMS_MIN_REVISION_WITH_LOW_CARDINALITY_TYPE 54405
 #define DBMS_MIN_REVISION_WITH_CLIENT_WRITE_INFO 54420
+#define DBMS_MIN_REVISION_WITH_CLIENT_ELAPSED_TIME 54444
```

Comment:
Should we bump `DBMS_TCP_PROTOCOL_VERSION`?

Reply:
Exactly! Done. Thanks.